gdc_ng_models/models/released_data.py

```diff
@@ -0,0 +1,91 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import validates
+from sqlalchemy.sql import schema, sqltypes
+
+from gdc_ng_models.models import audit
+
+Base = declarative_base()
+
+
+RELEASE_DATA_TYPE_VALUES = ["ssm", "cnv", "case"]
+
+
+class ReleasedDataMixin:
+    program_name = schema.Column(sqltypes.Text, nullable=False)
+    project_code = schema.Column(sqltypes.Text, nullable=False)
+    is_open = schema.Column(sqltypes.Boolean, nullable=False)
+
+    @property
+    def project_id(self):
+        return "{}-{}".format(self.program_name, self.project_code)
+
+    @validates("data_type")
+    def validate_data_type(self, key, data_type):
```

Comment:
should `data_type` be part of this then?

Reply:
oh hey good catch